packages/editor/src/store/selectors.js

```diff
@@ -439,7 +440,7 @@ export function isEditedPostSaveable( state ) {
  * @return {boolean} Whether post has content.
  */
 export function isEditedPostEmpty( state ) {
-	const blocks = getBlocksForSerialization( state );
+	const blocks = getBlocksForSerializationWithoutAttributes( state );
```

Comment:
For more clarity, maybe this could be refactored to return just the blockNames : `getBlocksNamesForSerialization`

Reply:
Changed. Also removed the export.